core/src/main/resources/error/error-classes.json

```diff
@@ -3008,11 +3008,6 @@
       "<message>. If necessary set <ansiConfig> to false to bypass this error."
     ]
   },
-  "_LEGACY_ERROR_TEMP_2001" : {
-    "message" : [
-      "<message> If necessary set <ansiConfig> to false to bypass this error"
-    ]
-  },
```

Comment:
Addressed the https://github.com/apache/spark/pull/38104#discussion_r987730351.

@MaxGekk FYI